src/modules/rdf/process.cpp

```diff
@@ -48,6 +48,12 @@ bool RDFModule::process(Dissolve &dissolve, ProcessPool &procPool)
      * multiple independent Configurations, we must loop over the specified targetConfigurations_ and calculate the partials
      * for each.
      */
+
+    // Check that Configurations have unmodified size factor
+    if (std::all_of(targetConfigurations_.begin(), targetConfigurations_.end(),
+                    [](const auto *cfg) { return cfg->appliedSizeFactor() != 1.0; }))
+        return true;
```

Comment:
Printing a message here would be good and informative, I think:
```suggestion
    if (std::all_of(targetConfigurations_.begin(), targetConfigurations_.end(),
                    [](const auto *cfg) { return cfg->appliedSizeFactor() != 1.0; }))
    {
        Messenger::print("One or more configurations have an applied size factor, so RDF calculation will be skipped.\n");
        return true;
    }
```